TimerCalculatorUtility/TimerCalculatorUtility/ViewModels/CalculatorViewModel.cs

```diff
@@ -0,0 +1,76 @@
+﻿
+using System;
+using System.ComponentModel;
+using System.Data;
+using System.Text;
+using Xamarin.Forms;
+
+namespace TimerCalculatorUtility.ViewModels
+{
+    public class CalculatorViewModel : INotifyPropertyChanged
+    {
+        private StringBuilder _readedString = new StringBuilder();
+
+        public CalculatorViewModel()
+        {
+            InputSymbolCommand = new Command<string>(AddSymbolToString);
+            ResultCommand = new Command(Resulting);
+            ClearCommand = new Command(Clear);
+            DleleteLastSymbolCommand = new Command(DeletelastSymbol);
+            PrecentCommand = new Command(CalcPrcent);
+        }
+
+        public string InputString { get; set; }
+        public string ResultString { get; set; }
+        public Command<string> InputSymbolCommand { get; set; }
```

Comment:
You aren't assigning commands except constructor. Commands properties can be made get-only